packages/browser/src/transports/utils.ts

```diff
@@ -86,7 +86,7 @@ export function getNativeFetchImplementation(): FetchImpl {
  * @param url report endpoint
  * @param body report payload
  */
-export function sendReport(url: string, body: string): void {
+export function sendReport(url: string, body: string | Uint8Array): void {
```

Comment:
`serializeEnvelope` now returns `string | Uint8Array`. In this case it will never actually be a `Uint8Array` because there's no attachment but both `sendBeacon` and `fetch` take `body: string | Uint8Array` so it seemed the best thing to do.